vms/job/templates/job/create.html

```diff
@@ -49,7 +49,7 @@
 							<p class="form-control-static" id="end_date_here">end_date</p>
 						</div>
 					</div>
-                    <script src="{% static "vms/js/update-event-dates.js" %}"></script>
+                    <script src="{% static 'vms/js/update-event-dates.js' %}"></script>
                     {% if form.name.errors %}
```

Comment:
@amruthasangeeth Why have we changed from double quote to single quote? Was this useful for you?